### PR TITLE
OSDOCS-4300: Docs for using Krew to manage plug-ins

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.12

Issue:
https://issues.redhat.com/browse/OSDOCS-4300

Link to docs preview:
https://53368--docspreview.netlify.app/openshift-enterprise/latest/cli_reference/openshift_cli/managing-cli-plugins-krew.html

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@kasturinarra @deejross This is ready for review whenever you get a chance. This is very basic, but I think that was the idea for 4.12. 

@gauravsingh85 If you can also please take a look. I marked this as tech preview, but let me know if we need to get any more explicit in stating that we're not doing any sort of support for actually using plug-ins installed by Krew yet.

Preview: https://53368--docspreview.netlify.app/openshift-enterprise/latest/cli_reference/openshift_cli/managing-cli-plugins-krew.html

File: _topic_maps/_topic_map.yml [hunk 1]
```diff
@@ -1077,7 +1079,7 @@ Topics:
   - Name: Installing the AWS Load Balancer Operator
     File: install-aws-load-balancer-operator
   - Name: Installing the AWS Load Balancer Operator on Secure Token Service cluster
-    File: installing-albo-sts-cluster  
+    File: installing-albo-sts-cluster
```
Comment (kasturinarra) [inline _topic_maps/_topic_map.yml:1082]: @bergerhoffer does this relate to this PR itself or is this something different ?

**Comment by kasturinarra** [comment]
@bergerhoffer one minor comment but rest looks good to me. May be we should get this reviewed through gaurav so that we do not miss anything obvious.

File: _topic_maps/_topic_map.yml (hunk 1 above)
Comment (bergerhoffer) [reply _topic_maps/_topic_map.yml:1082]: @kasturinarra This is just trailing whitespace that is automatically removed by my editor (other writers should also have this setting turned on, but as you can see, not everyone does :)

Comment (kasturinarra) [reply _topic_maps/_topic_map.yml:1082]: ah, got it now, thought you added the entire line. Never mind , thanks !!

File: modules/cli-krew-install-plugin.adoc [hunk 2]
```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_cli/installing-cli-plugins-krew.adoc
+
+:_content-type: PROCEDURE
+[id="cli-krew-install-plugin_{context}"]
+= Installing a CLI plug-in with Krew
+
+You can install a plug-in to the OpenShift CLI (`oc`) with Krew.
```
Comment (abhatt-rh) [inline modules/cli-krew-install-plugin.adoc:9]: Global comment: I was trying to find a better resource on prepositions to use with "install". I could not find specific guidance, yet a couple of examples from IBM Style under [Introducing the procedure](https://www.ibm.com/docs/en/ibm-style?topic=format-procedures#introducing-the-procedure), use _install something "on"_
```suggestion
You can install a plug-in on the OpenShift CLI (`oc`) with Krew.
```

**Review by abhatt-rh — COMMENTED** [review]
Hi @bergerhoffer ,
Nice work, LGTM! Added a minor suggestion for your consideration.

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: modules/cli-krew-install-plugin.adoc (hunk 2 above)
Comment (bergerhoffer) [reply modules/cli-krew-install-plugin.adoc:9]: @abhatt-rh thanks for the suggestion! Thinking about it more, I think that "for" is the most accurate word to use, so I will update to use this.

Based on some wording used in the other section on CLI plug-ins: https://53368--docspreview.netlify.app/openshift-enterprise/latest/cli_reference/openshift_cli/extending-cli-plugins.html#cli-installing-plugins_cli-extend-plugins

**Comment by gauravsingh85** [comment]
LGTM

**Pull request merged**